packages/core/src/components/popover/popover.tsx

```diff
@@ -652,7 +658,14 @@ export class Popover extends AbstractComponent<IPopoverProps, IPopoverState> {
 
 function ensureElement(child: React.ReactChild) {
```

Comment:
this could use a comment to explain that it's ultimately used to coerce children / goes through `understandChildren` into `render`